multiplayer/examples/sudoku/rollup.config.mjs

```diff
@@ -0,0 +1,15 @@
+import typescript from "@rollup/plugin-typescript"
```

Comment:
Since vite is our official template, wdyt about using that instead of the duplicate build setup?